src/installer/tests/HostActivation.Tests/FrameworkResolution/MultipleHives.cs

```diff
@@ -137,6 +139,7 @@ private List<FrameworkInfo> GetExpectedFrameworks(bool? multiLevelLookup)
             return expectedList;
         }
 
+        [SkipOnPlatform(TestPlatforms.OSX, "https://github.com/dotnet/runtime/issues/91486")]
```

Comment:
Can we narrow this to osx arm64? These should be running (and passing) on osx x64 CI builds currently.